src/main/webapp/profile.html

```diff
@@ -4,33 +4,179 @@
     <meta charset="UTF-8">
     <title>Profile</title>
     <link rel="stylesheet" href="profileStyle.css">
-    
+    <script src="profileScript.js"></script>
+    <script 
+      src="https://kit.fontawesome.com/3beeeddc2b.js" crossorigin="anonymous">
+    </script>
   </head>
 
   <body>
     <div id="content">
-      <div id="navigation-bar"> 
-        <div id="main-page-link"></div>
-        <div id="logout-link"></div>
+      <div id="navigation-bar">
+        <a href="index.html" id="main-page-link">
+          <i class="fas fa-globe-europe"></i>
+        </a>
+        <a href="index.html" id="logout-link">
+          <i class="fas fa-sign-out-alt"></i>
+        </a>
       </div>
+
       <div id="profile-picture">
-        <img src="./pictures/defaultProfilePicture.jpeg">
+        <div id="new-image">
+          <span class="fas fa-plus"></span>
+        </div>
+        <div id="avatar"><img src="./pictures/defaultProfilePicture.jpeg"></div>
+      </div>
+
+      <div id="friend-code">
+        <p id="text">Your friend code:</p>
+        <p id="friend-code-code">#AAAAA</p>
+      </div>
+
+      <div id="profile-info">
+        <div>name</div>
+        <form>
+          <input type="text" placeholder="New name">
+        </form>
+        <div>email</div>
+        <form>
+          <input type="text" placeholder="New mail">
+        </form>
+        <form>
+          <input type="radio" name="joke" value="bed" /> Send notifications
+          <input type="radio" name="joke" value="clock" /> Don't
```

Comment:
I think it would be better if the second choice was `Don't send notifications` and not just `Don't`